### ISSUE TITLE
probe library

### ISSUE THREAD

**Issue by samuelgarcia** [kwikteam/probes#1]
I have started a python package to deal with probes description here:
https://probeinterface.readthedocs.io/en/main/overview.html

I would like also to build a collections of probes based of the new proposed formats.
How did you build this collections of PRB ?

**Comment by rossant** [comment]
you should ask @neuromusic !

**Comment by samuelgarcia** [comment]
Hi @neuromusic,
 do you remember how you construct the PRB collections ?
Thanks a lot.

**Comment by neuromusic** [comment]
Indeed, I do. They were scraped from the database embedded in Neuronexus's software for their now-deprecated NiPOD.

Data dump and notebooks here: https://github.com/neuromusic/neuronexus-probe-data

**Comment by samuelgarcia** [comment]
Dear Justin.
Thank you for this.
I will have a look.

If you already were involved in probe geometry, and if you have time of course, maybe you could give a feedboack on our new probeinterface package:
https://probeinterface.readthedocs.io/en/main/overview.html
https://gin.g-node.org/spikeinterface/probeinterface_library

**Comment by samuelgarcia** [comment]
I look quickly into notebook.
Did you handle multi column design ?
In the notebook geting-the-geometries the x_locs were set to 0 for each shank, do you think we could reconstruct the absolute postion ?
